### ISSUE TITLE
Too thick line when tab groups are used

### ISSUE THREAD

**Issue by vanmash** [dracula/google-chrome#11]
When tab group is used the line around current tab is too thick (perhaps 1 or 2px) and it doesn't look smooth.

The look with default theme
![default](https://user-images.githubusercontent.com/2527832/111908789-e0e92e80-8a5a-11eb-8db0-bdd36d54bc17.PNG)
The look with Dracula theme
![Dracula](https://user-images.githubusercontent.com/2527832/111908794-e47cb580-8a5a-11eb-906b-687807382719.PNG)

Please check.

**Comment by nickimola** [comment]
Hi
Thanks for reporting, i'll check as soon as I can

**Comment by iam-cult** [comment]
Have this problem as well, on both Brave and Chrome. I think that this is caused by the gray line that separates the toolbar and the tabs, which seems to overlap the group colored line on background tabs, but does not overlap the colored line on active tabs
![Screen Shot 2021-03-23 at 09 43 56](https://user-images.githubusercontent.com/65368815/112156151-5b7d8f80-8bbc-11eb-9533-c8b83e39eab7.png)

**Comment by nickimola** [comment]
Hi
I've submitted for reviewing a new version (2.3) with a fix for this issue.
Wait for the update and let me know if it's solved.
Thanks

**Comment by iam-cult** [comment]
The original issue is solved, but I would suggest making the background tab color the same as the background color, as now the only way to tell the active tab is the purple bar, but the background tabs stand out unnecessarily from the background.

tl;dr: the tabs no longer look like tabs, just a bar with a purple line

**Comment by nickimola** [comment]
Ok, i'll open an improvement ticket for this and adjust as soon as I can.
Thanks for the feedback
